drivers/net/bnxt/bnxt_cpr.c

```diff
@@ -157,7 +157,8 @@ int bnxt_alloc_def_cp_ring(struct bnxt *bp)
 	cpr->cp_doorbell = bp->pdev->mem_resource[2].addr;
 	B_CP_DB_DISARM(cpr);
 	bp->grp_info[0].cp_fw_ring_id = cp_ring->fw_ring_id;
-	rc = bnxt_hwrm_func_cfg_def_cp(bp);
+	if (!BNXT_VF(bp))
+		rc = bnxt_hwrm_func_cfg_def_cp(bp);
```

Comment:
A default completion ring isn't just for the PF, the function should be modified to work with the VFs instead.